assets/css/style.css

```
@@ -6,41 +7,44 @@
 
 body {
     background-color: #d9dcd6;
+    font-family: 'Courier New', Courier, monospace;
```

```suggestion
    font-family: 'Gill Sans', 'Gill Sans MT', Calibri, 'Trebuchet MS', sans-serif;
```

I would set this as the default `font-family` and remove the `font-family` from `#main-panel`, `.benefits` and `header nav`. In other cases where you have to overwrite the `font-family` like `header` and `footer` you leave the other `font-family` there which is `'Trebuchet MS', 'Lucida Sans Unicode', 'Lucida Grande', 'Lucida Sans', Arial, sans-serif`